app/logic/stake_logic.rb

```diff
@@ -29,6 +29,8 @@ def get_stake_accounts
         p.payload[:config_urls]
       )
 
+      raise 'No results from `solana stakes`' if stake_accounts.blank?
```

Comment:
Consider creating custom exception, ie. `class NoResultsFromSolanaCli < StandardError; end` and raise it with proper message instead of raising generic RuntimeError